extensions/2.0/Vendor/EXT_mesh_manifold/README.md

```diff
@@ -0,0 +1,177 @@
+
+```json
+"meshes": [
+    {
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 3,
+            "TEXCOORD_0": 4
+          },
+          "mode": 4,
+          "material": 0,
+          "indices": 5
+        },
+        {
+          "attributes": {
+            "POSITION": 3,
+            "TEXCOORD_0": 4
+          },
+          "mode": 4,
+          "material": 1,
+          "indices": 6
+        }
+      ],
+      "extensions": {
+        "EXT_mesh_manifold": {
+          "manifoldPrimitive": {
+            "indices": 2,
+            "mode": 4,
+            "attributes": {
+              "POSITION": 3
+            }
+          },
+          "mergeIndices": 0,
+          "mergeValues": 1
+        }
+      }
+    }
+  ],
+"accessors": [
+    {
+      "name": "sparse indices",
+      "type": "SCALAR",
+      "componentType": 5125,
+      "count": 1728,
+      "bufferView": 2,
+      "byteOffset": 0
+    },
+    {
+      "name": "sparse values",
+      "type": "SCALAR",
+      "componentType": 5125,
+      "count": 1728,
+      "bufferView": 2,
+      "byteOffset": 6912
+    },
+    {
+      "name": "manifold indices",
+      "type": "SCALAR",
+      "componentType": 5125,
+      "count": 10320,
+      "bufferView": 3,
+      "byteOffset": 0,
+      "sparse": {
+        "count": 1728,
+        "indices": {
+          "bufferView": 2,
+          "byteOffset": 0,
+          "componentType": 5125
+        },
+        "values": {
+          "bufferView": 2,
+          "byteOffset": 6912
+        }
+      }
+    },
+    {
+      "name": "position",
+      "type": "VEC3",
+      "componentType": 5126,
+      "count": 2292,
+      "max": [
+        50,
+        50,
+        50
+      ],
+      "min": [
+        -50,
+        -50,
+        -50
+      ],
+      "bufferView": 4,
+      "byteOffset": 0
+    },
+    {
+      "name": "tex-coord",
+      "type": "VEC2",
+      "componentType": 5126,
+      "count": 2292,
+      "bufferView": 4,
+      "byteOffset": 12
+    },
+    {
+      "name": "first primitive indices",
+      "type": "SCALAR",
+      "componentType": 5125,
+      "count": 1728,
+      "bufferView": 3,
+      "byteOffset": 0
+    },
+    {
+      "name": "second primitive indices",
+      "type": "SCALAR",
+      "componentType": 5125,
+      "count": 8592,
+      "bufferView": 3,
+      "byteOffset": 6912
+    }
+  ],
+```
+
+## Usage
+
+While this extension places additional restrictions on how a mesh is stored, it is still a valid glTF 2.0 and thus will render properly even on software that does not implement support for this extension. For software that needs a manifold mesh rather than a rendering mesh, the simplest path is to simply replace the set of `primitives` with the `manifoldPrimitive`, which is still a valid glTF, but is simply missing all non-position `attributes`. 
```

Comment:
>  For software that needs a manifold mesh rather than a rendering mesh, the _simplest_ path is to _simply_ replace the set of `primitives` with the `manifoldPrimitive`, which is still a valid glTF, but is _simply_ missing all non-position `attributes`.

consider _simplifying_ this to

> Software that needs a manifold mesh rather than a rendering mesh **SHOULD** ignore the original mesh primitives and use the `manifoldPrimitive` instead.

Reply:
Good call - and I managed to remove another superfluous paragraph while I was at it.